### DISCUSSION TITLE
DLNA and/or upnp support in Audacious?

### DISCUSSION THREAD

**Discussion by juedan** [audacious-media-player/audacious#1497] [Q&A]
Hello,
in my home network I have setup an DLNA server which works perfect with my receiver, tablet, smartphone and TV. Now I want listen some music on my notebook with FreeBSD installed.
How to setup audacious for access to my DLNA server?
Thanks in advance for the answers
JueDan

**Comment by radioactiveman** [comment]
Audacious can play audio files from:
* file shares (SMB and FTP)
* network streams (HTTP(s))
* multimedia streams (MMS)

Look into these options and how to set them up on the server side.
Audacious as client needs no setup. Just make sure that the corresponding transport plugin is enabled.

**Reply by mariusZZ51** [reply]
"file shares (SMB) "   can you show me how because i couldn't manage to add all folders to player list !???
